eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/mapper/connection/ConnectionMapper.java

```diff
@@ -58,34 +58,34 @@ public List<ConnectionEntity> selectByClusterIdSinkTypeAndSinkIdAndCreateTimeRan
         @Param("startTime") Timestamp startTime, @Param("endTime") Timestamp endTime);
 
     @Options(useGeneratedKeys = true, keyProperty = "id", keyColumn = "id")
-    @Insert("INSERT INTO connection (cluster_id, source_type, source_id,"
-        + " sink_type, sink_id, runtime_id, status, topic, group_id, description)"
+    @Insert("INSERT INTO connection (cluster_id, source_type, source_id," + " sink_type, sink_id, runtime_id, status, topic, group_id, description)"
         + " VALUES ( #{clusterId}, #{sourceType}, #{sourceId}, "
```

Comment:
line break removed?

Reply:
mistakenly do this, I will fix it in future commits

Reply:
ok